### ISSUE TITLE
Haproxy Service Doesn't Start on Ubuntu 14.04

### ISSUE THREAD

**Issue by jgeorge300** [sous-chefs/haproxy#199]
I know Ubuntu 14.04 is not listed as a supported platform. But I tried to use it. The cookbook installs and configures haproxy successfully. But when you run the init script, it does not start the service. After reviewing the cookbook, I noticed it does not place the cookbook file 'haproxy-default' on the system and init start up remains disabled. Is there a reason 'haproxy-default' is no longer installed?

### Cookbook version
4.0.1

### Chef-client version
 12.18.31

### Platform Details
Ubuntu 14.04 (AWS)

### Scenario:
Install haproxy on a new instance, configure it and have the service running.

### Steps to Reproduce:
Using the cookbook as documented

### Expected Result:
haproxy running

### Actual Result:
haproxy doesn't start via init

**Comment by damacus** [comment]
@jgeorge300 would you mind pasting your config please?

**Comment by jgeorge300** [comment]
Contents of my recipe's default.rb:
```
haproxy_install 'package' do
end

haproxy_config_global '' do
  chroot '/var/lib/haproxy'
  daemon true
  maxconn 256
  log '/dev/log local0'
  log_tag 'WARDEN'
  pidfile '/var/run/haproxy.pid'
  stats_socket '/var/lib/haproxy/stats level admin'
  tuning 'bufsize' => '262144'
end

haproxy_config_defaults '' do
  mode 'http'
  timeout connect: '5000ms',
          client: '5000ms',
          server: '5000ms'
end

haproxy_frontend 'http-in' do
  bind '*:80'
  default_backend 'servers'
end

haproxy_backend 'servers' do
  server ['server1 127.0.0.1:8081 maxconn 32']
end

node.default['nexus3']['properties_variables'] = { port: '8081', args: '${jetty.etc}/jetty.xml,${jetty.etc}/jetty-http.xml,${jetty.etc}/jetty-requestlog.xml', context_path: '/' }

node.default['nexus3']['path'] = '/data'
node.default['nexus3']['home'] = '/data/nexus3'
node.default['nexus3']['data'] = '/data/sonatype-work/nexus3'

include_recipe 'nexus3'

cookbook_file '/etc/default/haproxy' do
  source 'haproxy-default'
  owner 'root'
  group 'root'
  mode '0644'
end

service 'haproxy' do
  action :restart
end
```

**Comment by rshade** [comment]
what's the output of haproxy -c /etc/haproxy/haproxy.cfg

**Comment by jgeorge300** [comment]
```
# haproxy -vc -f /etc/haproxy/haproxy.cfg
HA-Proxy version 1.4.24 2013/06/17
Copyright 2000-2013 Willy Tarreau <w@1wt.eu>

# cat /etc/haproxy/haproxy.cfg  
global
  user haproxy
  group haproxy
  log /dev/log local0
  log-tag WARDEN
  chroot /var/lib/haproxy
  daemon
  quiet
  stats socket /var/lib/haproxy/stats level admin
  stats timeout 2m
  pidfile /var/run/haproxy.pid
  tune.bufsize 262144

defaults
  timeout connect 5000ms
  timeout client 5000ms
  timeout server 5000ms
  log global
  mode http
  balance roundrobin
  option httplog
  option dontlognull
  option redispatch
  stats uri /haproxy-status


frontend http-in
  default_backend servers
  bind *:80
  maxconn 2000


backend servers
  server server1 127.0.0.1:8081 maxconn 32

```

**Comment by damacus** [comment]
Yeah this should be fixed by #200 Tested and works locally. Will push out a fix shortly.
